__generated__/globalTypes.ts

```diff
@@ -7,6 +7,13 @@
 // START Enums and Input Objects
```

Comment:
Move these types to [`src/subgraph/__generated__`](https://github.com/cryptonative-ch/mesa-interface/tree/dev/src/subgraph/__generated__)